release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1307,7 +1307,23 @@ As part of the continued deprecation of the Operator package manifest format, th
 
 *Console Kubevirt Plugin*
 
+* Previously, in a web console form that helped users add boot sources to virtualization templates, an explanatory text gave information only for Fedora, regardless of what operating system the template used. This update adds a fix that provides examples that are specific to the template's operating system so users have relevant guidance. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1908169[*BZ#1908169*])
 
+* Previously, in a web console wizard that helped users create virtual machine templates, imprecise language made it unclear whether an operation applied to the template or to a virtual machine. This fix clarifies the description so users can make an informed decision. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1922063[*BZ#1922063*])
+
+* In the web console, a vague error message caused unnecessary confusion for some users who tried to add a network interface to a virtual machine they were creating from a template. This update adds detail to the error message so users can troubleshoot the error more easily. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1924788[*BZ#1924788*])
```

Comment:
Do you want a _Previously_ here to be consistent?

Reply:
Thanks -- all changed as suggested.